web/templates/add_saved_snippet_modal.ts

```diff
@@ -0,0 +1,28 @@
+import {html, to_html} from "../shared/src/html.ts";
+import {$t} from "../src/i18n.ts";
+
+export default function render_add_saved_snippet_modal(context) {
+    return to_html(
+        html`<div id="add-new-saved-snippet-modal" class="new-style">
+            <form id="add-new-saved-snippet-form" class="new-style">
+                <label for="title" class="modal-field-label"
+                    >${$t({defaultMessage: "Title"})}</label
+                >
+                <input
+                    id="new-saved-snippet-title"
+                    type="text"
+                    name="title"
+                    class="modal_text_input saved-snippet-title"
+                    value=""
+                    autocomplete="off"
+                    spellcheck="false"
+                    autofocus="autofocus"
+                />
```

Comment:
I don't love how much less compact this is, but probably comparably readable? The only thing had a giant run-on line, this has a very tall list of unsorted attributes.

Can your tool sort the attributes? Seems like it might be a good idea if it doesn't make our life miserable with variables.

Reply:
I’m currently not touching the HTML myself (though I will need to in order to target JSX); HTML formatting is left to Prettier. Looks like there’s a [prettier-plugin-organize-attributes](https://www.npmjs.com/package/prettier-plugin-organize-attributes) that we should try.